### ISSUE TITLE
[TalkBack] Explicitly speak "X replaces Y" for text replacement

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#206]
```
What steps will reproduce the problem?
1. Talkback on
2. Enter input field (ex. mms)
3. Put a letter (ex. 'r')
4. Long click the EditText field and select the letter
5. Put another letter so replace the old one (ex. 't')

What is the expected output? What do you see instead?
I expected the feedback like "r replaced with t" but talkback only says "t".

What version of the product are you using? On what operating system?
- OS : Android JB
- Talkback version : 3.3.0

Please provide any additional information below.
As far as I know text replace scenario changed from v3.1.4-beta3 to 
v3.1.4-beta4.
(TextFormatters.java)
This scenario is necessary for Korean keyboard.
```

Original issue reported on code.google.com by `lriver1...@gmail.com` on 14 Mar 2013 at 6:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 2 Apr 2013 at 10:27
- Changed title: **[TalkBack] Explicitly speak "X replaces Y" for text replacement**
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.3.1_b1.
```

Original comment by `alanv@google.com` on 3 Apr 2013 at 1:00
- Changed state: **Fixed**
